space_dodge/drawing/pause_menu/settings.py

```
@@ -1,26 +1,24 @@
+import time
 
 import pygame
 import pygame_widgets
-import time
-
-from pygame_widgets.slider import Slider
-from pygame_widgets.textbox import TextBox
-
 from classes.button import Button
 from file_handling.constants_and_file_loading import (
     WINDOW, pause_background, muteImage, unmuteImage, PAUSE_FONT, pause_time, x_button_icon)
```

Ruff (0.11.9)

`file_handling.constants_and_file_loading.pause_time` imported but unused

Remove unused import: `file_handling.constants_and_file_loading.pause_time`

(F401)